demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -87,10 +90,13 @@ def is_valid_file(self, validate_rn: bool = True, skip_test_conf: bool = False)
             self.is_valid_description(beta_integration=False),
         ]
 
+        answer3 = []
+        if self.support_level in LEVEL_SUPPORT_OPTIONS_3:
+            answers3 = [self.is_valid_max_fetch_and_first_fetch()]
```

Comment:
you would have to comment to explain, also let's try to think on better var names